scripts/core/editor3/components/Editor3.jsx

```diff
@@ -173,9 +174,25 @@ export class Editor3Component extends React.Component {
         case 'soft-newline':
             newState = RichUtils.insertSoftNewline(editorState);
             break;
+        case 'delete':
+            if (!this.allowEditSuggestion('delete')) {
+                return 'handled';
+            }
+
+            if (suggestingMode) {
+                onCreateDeleteSuggestion('delete');
+                return 'handled';
+            }
+
+            newState = RichUtils.handleKeyCommand(editorState, command);
+            break;
         case 'backspace': {
+            if (!this.allowEditSuggestion('backspace')) {
```

Comment:
Are we allowing to edit suggestions when not in suggesting mode?

Reply:
Prevent to edit of suggestions in editing mode.